compiler/src/iree/compiler/GlobalOptimization/Passes.cpp

```diff
@@ -78,6 +77,7 @@ void buildGlobalOptimizationPassPipeline(
         return createFuseDequantizationMatmulPass(
             clEnableQuantizedMatmulReassociation);
       })
+      .addPass(createLiftGenericToTransposeBatchMatmulPass)
```

Comment:
I don't have full context. Do we want to scope the pass under a flag (e.g., `clEnableQuantizedMatmulReassociation` or `transformOptions.options.dataTiling`)?

If it is a general optimization, we can keep it here. If not, we can probably control it under a `data-tiling`.